### PR TITLE
Refactor to es6 standards

### PR DESCRIPTION
**Pull request by nunofgs**
Depends on #30.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,37 +4,49 @@
   "description": "Koa Pagination",
   "license": "MIT",
   "author": "Seegno",
-  "main": "index.js",
+  "main": "dist/src/index.js",
```
Comment (franciscocardoso) [inline package.json]: `dist/src`?

File: package.json [hunk 2]
```diff
@@ -4,37 +4,50 @@
   "description": "Koa Pagination",
   "license": "MIT",
   "author": "Seegno",
-  "main": "index.js",
+  "main": "dist/src",
   "repository": "git@github.com:seegno/koa-pagination",
+  "options": {
+    "mocha": "--compilers js:babel-register --recursive --require co-mocha --require should"
+  },
   "scripts": {
```
Comment (franciscocardoso) [inline package.json]: Suggestion:

```js
  "scripts": {
    "changelog": "github_changelog_generator --no-issues --header-label='# Changelog' --future-release=v$npm_config_future_release && sed -i '' -e :a -e '$d;N;2,4ba' -e 'P;D' CHANGELOG.md",
    "cover": "babel-node node_modules/.bin/isparta cover --report html _mocha -- $npm_package_options_mocha",
    "lint": "eslint src test && jscs src test",
    "prepublish": "npm run transpile",
    "test": "mocha $npm_package_options_mocha",
    "transpile": "rm -rf dist/* && babel src --out-dir dist/src",
    "version": "npm run changelog --future-release=$npm_package_version && npm run transpile && git add -A CHANGELOG.md dist"
  }
```

File: package.json [hunk 3]
```diff
@@ -4,37 +4,50 @@
   "description": "Koa Pagination",
```
Comment (franciscocardoso) [inline package.json]: Please run `sort-package-json`.

**Comment by franciscocardoso** [comment]
We should update the README.

File: src/index.js [hunk 4]
```diff
@@ -3,38 +3,30 @@
...
  * Export `PagerMiddleware`.
  */
 
-module.exports = function(options) {
-  options = _.assign({
-    maximum: 50,
-    unit: 'items'
-  }, options);
-
+export default function ({ maximum = 50, unit = 'items' } = {}) {
```
Comment (franciscocardoso) [inline src/index.js]: Extra space after `function` keyword?

File: src/index.js [hunk 5]
```diff
@@ -3,38 +3,30 @@
...
  * Export `PagerMiddleware`.
  */
 
-module.exports = function(options) {
-  options = _.assign({
-    maximum: 50,
-    unit: 'items'
-  }, options);
-
+export default function({ maximum = 50, unit = 'items' } = {}) {
```
Comment (ruimarinho) [inline src/index.js]: @nunofgs what about adding the default exports plugin?

File: package.json [hunk 6]
```diff
@@ -4,37 +4,50 @@
...
+    "test": "NODE_ENV=test mocha $npm_package_options_mocha",
+    "transpile": "rm -rf dist/* && babel src --out-dir dist/src",
+    "version": "npm run changelog --future-release=$npm_package_version && npm run transpile && git add -A CHANGELOG.md dist"
   },
   "dependencies": {
     "debug": "2.2.0",
     "http-content-range-format": "1.0.0",
-    "is-safe-integer": "^1.0.1",
-    "lodash": "4.12.0",
+    "is-safe-integer": "1.0.1",
```
Comment (ruimarinho) [inline package.json]: Not needed as node 4 (which this package requires) already ships `Number.isSafeInteger`.

Comment (ruimarinho) [inline package.json]: cc @nunofgs

File: package.json [hunk 7]
```diff
@@ -4,37 +4,50 @@
...
-    "test": "NODE_ENV=test mocha --recursive",
-    "version": "npm run changelog --future-release=$npm_package_version && git add -A CHANGELOG.md"
+    "lint": "eslint src test && jscs src test",
+    "prepublish": "npm run transpile",
+    "test": "NODE_ENV=test mocha $npm_package_options_mocha",
+    "transpile": "rm -rf dist/* && babel src --out-dir dist/src",
+    "version": "npm run changelog --future-release=$npm_package_version && npm run transpile && git add -A CHANGELOG.md dist"
   },
   "dependencies": {
     "debug": "2.2.0",
```
Comment (ruimarinho) [inline package.json]: Should be using debugnyan?

File: test/request.js [hunk 8]
```diff
@@ -3,32 +3,29 @@
  * Module dependencies.
```
Comment (ruimarinho) [inline test/request.js]: Remove file?

**Pull request merged**